packages/kbn-dev-utils/src/worker/index.ts

```diff
@@ -16,23 +16,29 @@ import { observeLines } from '@kbn/stdio-dev-helpers';
 
 // import type { Result } from './kibana_worker';
 
-interface StartTSWorkerArgs {
+interface StartTSWorkerArgs extends ForkOptions {
```

Comment:
Just decided to include all `ForkOptions` for this util, not strictly needed, but seemed simpler.